### PR TITLE
Add brass sbh method function

### PR DESCRIPTION
**Pull request by chacalle**
## Describe changes

Add simple/quick implementation of the Brass indirect sbh method.

You can skim Chapter III Section A and B if you want more quick background about the method. Starts on page 101 of the pdf https://unstats.un.org/unsd/demographic/standmeth/handbooks/Manual_X-en.pdf

This manual gives great example test data and the expected results that I used for a package test.

There are some other small changes in here to get the package checks working again.

## Checklist

* [X] Have you read the [contributing guidelines](https://github.com/ihmeuw-demographics/packageTemplate/wiki#guide-to-r-package-development) for `ihmeuw-demographics` R packages?
* [X] Have you successfully run `devtools::check()` locally?
* [X] Have you updated or added function (and vignette if applicable) documentation? Did you update the 'man' and 'NAMESPACE' files with `devtools::document()`?
* [X] Have you added in tests for the changes included in the PR?
* [X] Do the changes follow the `ihmeuw-demographics` [code style](https://github.com/ihmeuw-demographics/packageTemplate/wiki/Code-style-guide)?
* [X] Do the changes require updates to other repositories which use this package? If yes, make the necessary updates in those repos, and consider integration tests for those repositories.

## Details of PR

Jeff included similar code in an old branch of demogsurv but it works with microdata. I think we will eventually want a simple function that works with tabulated data since we may only have tabulations from a lot of censuses / old surveys.

https://github.com/mrc-ide/demogsurv/blob/u5mr-indirect/R/child-mortality-indirect.R

### CODE REVIEW

File: .github/workflows/check-standard.yaml [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
-# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches:
-      - master
+    branches: [main, master]
```
Comment (erinamay) [inline .github/workflows/check-standard.yaml:5]: why keep master in here?

File: R/sbh.R [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  coeffs <- sbh_trussell_coeffs[model == model_schedule]
+
+  assertive::assert_is_a_string(n_women_col)
+  assertive::assert_is_a_string(ceb_col)
+  assertive::assert_is_a_string(ced_col)
+
+  # check `id_cols` argument
+  age_cols <- c("age_start", "age_end")
+  assertive::assert_is_character(id_cols)
+  error_msg <- paste0("`id_cols` must include '",
```
Comment (erinamay) [inline R/sbh.R:86]: for consistency should this be:
paste0(
...
)
?

**Review by erinamay — APPROVED** [review]

**Comment by chacalle** [comment]
Resolved remaining pkgdown build issues in separate branch to prevent this PR from getting even bigger... https://github.com/ihmeuw-demographics/demCore/pull/84

File: .github/workflows/check-standard.yaml (hunk 1 above)
Comment (chacalle) [reply .github/workflows/check-standard.yaml:5]: I updated these files with code like `usethis::use_github_action("check-standard")` that automatically updates to the current best R practice in https://github.com/r-lib/actions/tree/v2/examples

**Pull request merged**